### PR TITLE
Add SDK samples from rel/3.4

### PR DESCRIPTION
**Pull request by nohwnd**
Add samples that were added only to 3.4 branch. 

Related to https://github.com/microsoft/testfx/pull/3073

### CODE REVIEW

**Comment by nohwnd** [comment]
For your review @evangelink, I don't know if more changes are needed on main that you only put to 3.4.

**Comment by cremor** [comment]
Seems like the deletion of the Usings.cs files is missing here.

File: samples/public/Directory.Build.props [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <Project>
 
   <PropertyGroup>
+    <AspireTestingVersion>8.0.1</AspireTestingVersion>
     <CodeCoverageVersion>17.10.4</CodeCoverageVersion>
-    <MSTestVersion>3.4.0</MSTestVersion>
+    <MSTestVersion>3.4.1</MSTestVersion>
```
Comment (Evangelink) [inline samples/public/Directory.Build.props]:
```suggestion
    <MSTestVersion>3.4.3</MSTestVersion>
```

**Review by Evangelink — APPROVED** [review]
Oh thanks @nohwnd, I definitely forgot to port the change when I fixed the branch.

**Pull request merged**